insonmnia/worker/server.go

```diff
@@ -1014,6 +1014,11 @@ func (m *Worker) taskAllowed(ctx context.Context, request *sonm.StartTaskRequest
 		return false, ref, err
 	}
 	if level < m.cfg.Whitelist.PrivilegedIdentityLevel {
+		volumes := request.GetSpec().GetContainer().GetVolumes()
+		mounts := request.GetSpec().GetContainer().GetMounts()
+		if (volumes != nil && len(volumes) > 0) || (mounts != nil && len(mounts) > 0) {
```

Comment:
```suggestion
		if len(volumes) > 0 || len(mounts) > 0 {
```
Since `len(nil)` gives 0.